specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -16,119 +17,119 @@ enum TextCategory {
+  severity?: int32;
 }
 
-@doc("The analysis request of the image.")
+@doc("The image analysis request.")
 model AnalyzeImageOptions {
   @doc("The image needs to be analyzed.")
   image: ImageData;
 
-  @doc("The categories will be analyzed. If not assigned, a default set of the categories' analysis results will be returned.")
+  @doc("The categories will be analyzed. If they are not assigned, a default set of analysis results for the categories will be returned.")
   categories?: ImageCategory[];
+
+  @doc("This refers to the type of image analysis output. If no value is assigned, the default value will be \"FourSeverityLevels\".")
+  outputType?: AnalyzeImageOutputType = AnalyzeImageOutputType.FourSeverityLevels;
 }
 
-@doc("The content or blob url of image, could be base64 encoding bytes or blob url. If both are given, the request will be refused. The maximum size of image is 2048 pixels * 2048 pixels, no larger than 4MB at the same time. The minimum size of image is 50 pixels * 50 pixels.")
+@doc("The image can be either base64 encoded bytes or a blob URL. You can choose only one of these options. If both are provided, the request will be refused. The maximum image size is 2048 x 2048 pixels and should not exceed 4 MB, while the minimum image size is 50 x 50 pixels.")
 @projectedName("csharp", "ContentSafetyImageData")
 model ImageData {
-  @doc("Base64 encoding of image.")
+  @doc("The Base64 encoding of the image.")
   content?: bytes;
 
-  @doc("The blob url of image.")
+  @doc("The blob url of the image.")
   blobUrl?: url;
 }
 
-@doc("The analysis response of the image.")
+@doc("The image analysis response.")
 model AnalyzeImageResult {
-  @doc("Analysis result for Hate category.")
-  hateResult?: ImageAnalyzeSeverityResult;
-
-  @doc("Analysis result for SelfHarm category.")
-  selfHarmResult?: ImageAnalyzeSeverityResult;
-
-  @doc("Analysis result for Sexual category.")
-  sexualResult?: ImageAnalyzeSeverityResult;
-
-  @doc("Analysis result for Violence category.")
-  violenceResult?: ImageAnalyzeSeverityResult;
+  @doc("Analysis result for categories.")
+  categoriesAnalysis: ImageCategoriesAnalysis[];
 }
 
 @doc("Image analysis result.")
-model ImageAnalyzeSeverityResult {
-  @doc("The image category.")
+model ImageCategoriesAnalysis {
+  @doc("The image analysis category.")
   category: ImageCategory;
 
-  @doc("The higher the severity of input content, the larger this value, currently its value could be: 0,2,4,6.")
-  severity: int32;
+  @doc("The value of this field is determined by the output type specified in the request. The output type could be ‘FourSeverityLevels’ or ‘EightSeverity Levels’, and the output value can be 0, 2, 4, 6 or 0, 1, 2, 3, 4, 5, 6, or 7. The value increases with the severity of the input content.")
```

Comment:
Same comment as above.
```suggestion
  @doc("The value increases with the severity of the input content. The value of this field is determined by the output type specified in the request. The output type could be ‘FourSeverityLevels’ or ‘EightSeverity Levels’, and the output value can be 0, 2, 4, 6 or 0, 1, 2, 3, 4, 5, 6, or 7.")
```

Reply:
Make sense! Fixed according to your suggestion.